src/scss/views/widgets/_avatar.scss

```diff
@@ -0,0 +1,19 @@
+.avatar-wrapper {
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
didn't wanna use `flex`?

Reply:
I think it'll make the wrapper 100% width across by default if I do

Reply:
cool, just 🔪 the `vertical-align` rule if it's not doing nothin'!